test-integration/Microsoft.Azure.WebJobs.Extensions.OpenApi.Document.Tests/Post_AppicationJson_Decimal_Tests.cs

```diff
@@ -0,0 +1,114 @@
+{
+    [TestClass]
+    [TestCategory(Constants.TestCategory)]
+    public class Post_AppicationJson_Decimal_Tests
+    {
+        private static HttpClient http = new HttpClient();
+
+        private JObject _doc;
+
+        [TestInitialize]
+        public async Task Init()
+        {
+            var json = await http.GetStringAsync(Constants.OpenApiDocEndpoint).ConfigureAwait(false);
+            this._doc = JsonConvert.DeserializeObject<JObject>(json);
+        }
+
+        [DataTestMethod]
+        [DataRow("/post-applicationjson-decimal", "post")]
+        public void Given_OpenApiDocument_Then_It_Should_Return_OperationRequestBody(string path, string operationType)
+        {
+            var requestBody = this._doc["paths"][path][operationType]["requestBody"];
+
+            requestBody.Should().NotBeNull();
+        }
+
+        [DataTestMethod]
+        [DataRow("/post-applicationjson-decimal", "post", "text/plain")]
+        public void Given_OpenApiDocument_Then_It_Should_Return_OperationRequestBodyContentType(string path, string operationType, string contentType)
+        {
+            var content = this._doc["paths"][path][operationType]["requestBody"]["content"];
+
+            content[contentType].Should().NotBeNull();
+        }
+
+        [DataTestMethod]
+        [DataRow("/post-applicationjson-decimal", "post", "text/plain", "number", "decimal")]
+        public void Given_OpenApiDocument_Then_It_Should_Return_OperationRequestBodyContentTypeSchema(string path, string operationType, string contentType, string propertyType, string propertyFormat)
+        {
+            var content = this._doc["paths"][path][operationType]["requestBody"]["content"];
+
+            var value = content[contentType]["schema"];
+
+            value.Should().NotBeNull();
+            value.Value<string>("type").Should().Be(propertyType);
+            value.Value<string>("format").Should().Be(propertyType);
+        }
+
+        [DataTestMethod]
+        [DataRow("/post-applicationjson-decimal", "post", "200")]
+        public void Given_OpenApiDocument_Then_It_Should_Return_OperationResponse(string path, string operationType, string responseCode)
+        {
+            var responses = this._doc["paths"][path][operationType]["responses"];
+
+            responses[responseCode].Should().NotBeNull();
+        }
+
+        [DataTestMethod]
+        [DataRow("/post-applicationjson-decimal", "post", "200", "application/json")]
+        public void Given_OpenApiDocument_Then_It_Should_Return_OperationResponseContentType(string path, string operationType, string responseCode, string contentType)
+        {
+            var content = this._doc["paths"][path][operationType]["responses"][responseCode]["content"];
+
+            content[contentType].Should().NotBeNull();
+        }
+
+        [DataTestMethod]
+        [DataRow("/post-applicationjson-decimal", "post", "200", "application/json", "decimalObjectModel")]
```

Comment:
Same problem with "decimalObjectModel". your app wrote your model type name "decimalTypeObjectModel". Please change this.

Reply:
thanks i fixed it